podcasts/NotificationsViewController.swift

```diff
@@ -47,6 +47,25 @@ class NotificationsViewController: PCViewController, UITableViewDataSource, UITa
                 return L10n.notificationsPocketCastOffers
             }
         }
+
+        var value: Bool {
+            switch self {
+            case .newEpisodes:
+                return false
+            case .podcastsChosen:
+                return false
+            case .appBadges:
+                return false
+            case .trendingRecommendations:
+                return false
+            case .dailyReminders:
+                return Settings.notificationsOnboardingTips
+            case .newFeaturesAndTips:
+                return false
+            case .pocketCastsOffers:
+                return false
+            }
+        }
```

Comment:
I think collapsing the falses would make skimming a bit easier.

```suggestion
        var value: Bool {
        case .dailyReminders:
            return Settings.notificationsOnboardingTips
        case .newEpisodes,
                 .podcastsChosen, 
                 .appBadges,
                 .trendingRecommendations,
                 .newFeaturesAndTips,
                 .pocketCastsOffers:
            return false
            }
        }
```